src/tv2-common/parts/server.ts

```diff
@@ -232,6 +233,8 @@ function getServerSelectionBlueprintPiece(
 		metaData: {
 			type: Tv2PieceType.VIDEO_CLIP,
 			outputLayer: Tv2OutputLayer.PROGRAM,
+			audioMode:
+				layers.SourceLayer.SelectedServer === SharedSourceLayer.SelectedVoiceOver ? Tv2AudioMode.VOICE_OVER : undefined,
```

Comment:
Whats the reasoning behind these being set to `undefined` when `.SelectedVoiceOver` is false?

Just wondering if they are suppose to be set to `Tv2AudioMode.FULL` like EVS was. I don't know enough to say what it correct, so i though i'd place the question, and others might be able to answer.

Reply:
Refer to this comment chain https://github.com/tv2/sofie-blueprints-inews/pull/227#discussion_r1404002026

Reply:
It should also not be `undefined` here